components/Slider/src/MDCSlider.m

```diff
@@ -209,6 +213,25 @@ - (UIColor *)backgroundTrackTickColorForState:(UIControlState)state {
   return color;
 }
 
+- (void)setThumbRadius:(CGFloat)thumbRadius forState:(UIControlState)state {
+  NSAssert(thumbRadius >= 0, @"Thumb radius must be a positive value");
+  float radius = (float)thumbRadius;
+  _thumbRadiusesForState[@(state)] = [NSNumber numberWithFloat:radius];
+  if (self.state == state) {
+    [self updateThumbForState];
+  }
+}
+
+- (CGFloat)thumbRadiusForState:(UIControlState)state {
+  if ([_thumbRadiusesForState objectForKey:@(state)]) {
+    return [_thumbRadiusesForState[@(state)] floatValue];
```

Comment:
Please use `doubleValue` and cast to CGFloat.

Reply:
Updated the API to use a `float` instead of a `double` or `CGFloat`.